src/main/java/org/mifos/connector/mockPaymentSchema/api/BatchApi.java

```diff
@@ -0,0 +1,28 @@
+package org.mifos.connector.mockPaymentSchema.api;
+
+import org.mifos.connector.mockPaymentSchema.schema.AuthorizationRequest;
+import org.mifos.connector.mockPaymentSchema.service.BatchService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class BatchApi {
+
+    @Autowired
+    private BatchService batchService;
+
+    @PostMapping("/batches/{batchId}")
```

Comment:
this doesn't follow API guidelines.
A POST API usually doesn't have id in path

Reply:
@fynmanoj this is resolved. Would be adding check for command = authorize in the method.

Reply:
command is used